Car_owners/application/views.py

```diff
@@ -240,3 +253,17 @@ def get_swagger_parameters() -> dict[str, list[openapi.Parameter]]:
     @swagger_auto_schema(**get_swagger_parameters())
     def list(self, request: request_type, *args, **kwargs) -> response_type:
         return super().list(request, *args, **kwargs)
+
+    @action(detail=False, name="unrepaired")
+    def unrepaired(self, request, *args, **kwargs):
+        """
+        Endpoint listed all unrepaired cars.
+        """
+        # Additional request validation
+        if response := self.request_validation(request):
```

Comment:
I don't understand this `if`, `request_validation` always returns a response not if I'm not mistaken. Maybe you need to add an exception raise there?

Reply:
possibly all requests should be validated by default? Not only for this endpoint

Reply:
The `request_validation` method returns `Response` only, if there was something wrong with the `request`, otherwise it returns `None`. That is the reason, why `if` statement is needed - if there was wrong data, the `Response` is passed to the `return` statement of the action method (list or unrepaired) and the message is shown - otherwise nothing happen and action method runs normally.

Reply:
ok you're right about the method returning value, I misread the code

Reply:
but I will leave it up to you if you want to implement it now or leave a comment in code with # TODO for the future

Reply:
Ok, I will keep it in mind and think about it maybe at another branch.